test/test_cache.py

```diff
@@ -33,7 +33,8 @@ def setUp(self):
             hidden=date(2000, 1, 1),
             attributes={"onehour": True, "kids": False, "available": True},
             summary="text",
-            description="long text",
+            description="engelmz",
+            description_html="<b><font color="red">engelmz</font></b>",
```

Comment:
This is invalid syntax and wrongly repeated twice below. AFAIK the `black` formatter prefers `'<b><font color="red">engelmz</font></b>'` in these cases.

Reply:
Pretty sure I just fixed it.